build-scripts/ocp-artifacts/3.11-plashet-to-ocp-artifacts.sh

```diff
@@ -26,7 +26,7 @@ fi
 FULL_VERSION="${VERSION}-${RELEASE}"
 
 BUILD_MODE="${3}"
-BASEDIR="/mnt/rcm-guest/puddles/RHAOS"
+BASEDIR="/mnt/data/pub/RHOCP"
```

Comment:
I doubt this script is still needed.

Historically, this script is used to push files on rcm-guest to the old mirror.openshift.com. It prepares some symlinks and eventually runs `push.enterprise.sh` on rcm-guest. After mirror.openshift.com was moved to S3, we deleted related code but code including preparing symlinks (http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/plashets/3.11/{latest|no-image-latest}) is still remained.

I saw no real use of those symlinks in our pipeline. In case people from other teams are still using them, we should redirect them to ocp-artifacts anyway.

So I would suggest removing this script.

Reply:
Fixed in e8db8cea361bb82124bee6bf6db92f44c1e7736e